doc/source/reference/arrays.scalars.rst

```diff
@@ -65,19 +65,20 @@ Some of the scalar types are essentially equivalent to fundamental
 Python types and therefore inherit from them as well as from the
 generic array scalar type:
 
-====================  ================================
-Array scalar type     Related Python type
-====================  ================================
-:class:`int_`         :class:`IntType` (Python 2 only)
-:class:`float_`       :class:`FloatType`
-:class:`complex_`     :class:`ComplexType`
-:class:`bytes_`       :class:`BytesType`
-:class:`unicode_`     :class:`UnicodeType`
-====================  ================================
+====================  ===================  =============
+Array scalar type     Related Python type  Inherits?
+====================  ===================  =============
+:class:`int_`         :class:`int`         Python 2 only
+:class:`float_`       :class:`float`       yes
+:class:`complex_`     :class:`complex`     yes
+:class:`bytes_`       :class:`bytes`       yes
+:class:`unicode_`     :class:`unicode`     yes
```

Comment:
```suggestion
:class:`str_`         :class:`str`         yes
```
Considering `unicode` was removed in python 3 might it not be better to link to `str` instead?

Reply:
Good catch